software/template/config.h

```diff
@@ -0,0 +1,20 @@
+typedef int32_t REAL;
+static const uint8_t MAX_DUTY_CYCLE = 90u;
+
+// If defined, use a PID
+// #define PID
+
+static const REAL Kp = (REAL)10.0;
+static const REAL Ki = (REAL)0.0;
+static const REAL Kd = (REAL)0.5;
```

Comment:
What's going on here? wouldn't this just truncate it to 0? REAL is an integer type?

Reply:
I set it to an integer type for now because we're just doing `*10` anyway - will add a comment to elaborate.